WordPressKit/SiteDesignServiceRemote.swift

```diff
@@ -1,35 +1,22 @@
 import Foundation
 import WordPressShared
 
-public struct SiteDesignRequest: Encodable {
-    public let previewSize: CGSize?
-    public let scale: CGFloat?
-
-    public init(previewSize: CGSize? = nil, scale: CGFloat? = nil) {
-        self.previewSize = previewSize
-        self.scale = scale
-    }
-
-    public func asParameters() -> [String: AnyObject] {
-        var parameters: [String: AnyObject] = [:]
-
-        if let previewWidth = previewSize?.width {
-            parameters["preview_width"] = previewWidth as AnyObject
-        }
-
-        if let scale = scale {
-            parameters["scale"] = scale as AnyObject
-        }
-
-        return parameters
+public struct SiteDesignRequest {
+    public let parameters: [String: AnyObject]
+
+    public init(withThumbnailSize thumbnailSize: CGSize) {
+        parameters = [
+            "preview_width": "\(thumbnailSize.width)" as AnyObject,
+            "scale": UIScreen.main.nativeScale as AnyObject
```

Comment:
Just adding a note here to capture the requirement shift. 
we'll want to expose a way to add a way to set `"group": "original,single-page"` to this. 

https://github.com/wordpress-mobile/gutenberg-mobile/issues/3125#issuecomment-784230338

Reply:
Added via https://github.com/wordpress-mobile/WordPressKit-iOS/pull/352/commits/00700a6d53e54a320333f4ea7b0a0929869bb431, and made available via interface to WordPress-iOS.